src/github.com/getlantern/flashlight/server/server.go

```diff
@@ -138,93 +135,6 @@ func (server *Server) ListenAndServe() error {
 	return fs.Serve(l)
 }
 
```

Comment:
This function was moved to `nattywad.go` and `nattywad_stubs.go` so it can be disabled on android hosts based on build tags.